source/turtles/turtle_hero.py

```diff
@@ -60,19 +66,31 @@ def move_down(self, pixels):
 
 class TurtleHero(Turtle):
     def __init__(self, type, size_coeff, name, position):
+        self.image_sheet = pygame.image.load('D:\\Users\\Ola\\PycharmProjects\\turtle_game\\source\\turtles\\sv_turtle_sheet.png').convert_alpha()
         super().__init__(type, size_coeff, name, position)
         self.ACC = 600
         self.TICK = 1 / 60.0
         self.SPEED_FAST = 300
         self.SPPED_SLOW = 150
-        self.is_jumping = JumpStates.IDLE #jumping state machine: 0 - no jumping, 1 - jump up, 2 - jump down
+        self.is_jumping = JumpStates.IDLE# jumping state machine: 0 - no jumping, 1 - jump up, 2 - jump down
```

Comment:
Hmm, that comment is a bit confusing, since the *jump_states* are now being treated as enums. Delete it.